### ISSUE TITLE
kubefwd formula installing xn2 instead of kubefwd

### ISSUE THREAD

**Issue by mbuccini** [txn2/homebrew-tap#1]
Hello, 

I would like to report that since a few days (https://github.com/txn2/homebrew-tap/commits/master/Formula/kubefwd.rb) it seems that the `kubefwd` formula is not installing anymore `kubefwd` but `xn2`. I just tried on my mac to install it with the command:

```brew install txn2/tap/kubefwd```

and all I see after installing it is 

```✔ /usr/local/Cellar/kubefwd/1.0.2/bin
14:55 $ ls -l
total 28648
-r-xr-xr-x  1 user  admin  14666344 Nov 19 21:13 xn2
```

**Comment by cjimti** [comment]
@mbuccini thank you for catching that. I'll get that fixed asap.

**Comment by cjimti** [comment]
Fixed and re-packaged in 1.4.8. Thank you for reporting this.
